torchao/ops.py

```diff
@@ -592,3 +593,31 @@ def _(
     bias: Tensor,
 ) -> Tensor:
     return input_scale.new_empty(*input.shape[:-1], weight.shape[0])
+
+
+def mx_fp8_bf16(A: Tensor, B: Tensor, A_scale: Tensor, B_scale: Tensor):
+    """Defines a matmul between two fp8 tensors w/ MX scales in E8MO and returns a bf16 tensor.
```

Comment:
nit: should we clarify this only supports `float8_e4m3fn` for now?  

If we wanted to add `float8_e5m2` or the `nuz` variants, would it also go in this function?